addons/web/static/src/scss/popover.scss

```diff
@@ -52,12 +52,12 @@
     $arrow-pad: 8px;
     $arrow-size: 8px;
     $animation-time: 0.2s;
-    width: 280px;
     max-width: 100vw;
 
     position: fixed;
     top: 0; // will be updated in js
     left: 0; // but let's set it in the viewport in case of test/debug
+    z-index: 1060;
```

Comment:
we should maybe have a variable for this

Reply:
Agree, there was with bootstrap: `$zindex-popover: 1060 !default;`

We can either use it again or copy/paste it somewhere and prefix it with `-o`.